.github/ISSUE_TEMPLATE/tfaga-issue.yml

```diff
@@ -0,0 +1,30 @@
+name: Generic testing-farm-as-github-action issue template
+description: File a generic issue here
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to report this issue to us. We appreciate the
```

Comment:
Yes I've looked at the preview. I like when the lines are shorter :). But I will change the formatting and we will see. Maybe it will be better that way.

Reply:
Do you like it more now?

Reply:
I think it looks better now.